scripts/db.out.ogr/db.out.ogr.py

```diff
@@ -67,7 +67,7 @@ def main():
     layer = options["layer"]
     format = options["format"]
     output = options["output"]
-    table = options["table"]
+    table = options["table"]  # noqa: F841
```

Comment:
I looked at it and this looks like option table was never used, so I think we should remove it in a separate PR. It would sort of break backwards compatibility, but if the option was never used, I assume it would be fine.